libpod/oci_conmon_exec_common.go

```diff
@@ -745,34 +744,9 @@ func prepareProcessExec(c *Container, options *ExecOptions, env []string, sessio
 		pspec.User = processUser
 	}
 
-	ctrSpec, err := c.specFromState()
-	if err != nil {
-		return nil, err
-	}
-
-	allCaps, err := capabilities.BoundingSet()
-	if err != nil {
+	if err := setProcessCapabilitiesExec(c, options, user, execUser, pspec); err != nil {
```

Comment:
I would prefer this to me.
c.setProcessCapabilitiesExec(options, user, execUser, pspec)

Reply:
Done. I also changed the signature of prepareProcessExec for consistency.